packages/core/chains/package.json

```diff
@@ -1,4 +1,4 @@
 {
-  "main": "dist/wagmi-core-chains.cjs.js",
-  "module": "dist/wagmi-core-chains.esm.js"
+  "main": "dist/wagmi-core-chains.esm.js",
```

Comment:
Ideally, we don't even need these `package.json`'s, but we may need them for a couple of reasons:

- Some tooling (Jest) don't respect the `exports` field, and
- Setting `moduleResolution` to anything other than `node16` or `nodenext` in `tsconfig.json` breaks type resolution. However, if we do set `moduleResolution` to one of those values, we need to append the `.js` extension to every relative import (yuck). There are long running issues for this [here](https://github.com/microsoft/TypeScript/issues/50152) and [here](https://github.com/microsoft/TypeScript/issues/49083).